### DISCUSSION TITLE
Startup errors using new image on raspberry pi

### DISCUSSION THREAD

**Discussion by itpick** [dani-garcia/vaultwarden#3271] [Q&A]
Had to update to latest image since the client wasn't working with my setup.  I haven't used my instance in probably a year due to moving around.

I have a rapsberry pi 4 running rasbian buster.

"Operation not permitted" }', library/std/src/sys/unix/time.rs:350:72
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: Os { code: 1, kind: PermissionDenied, message: "Operation not permitted" }', library/std/src/sys/unix/time.rs:350:72

Anyone know a way to workaroudn this to fix?

### Deployment environment

* vaultwarden version:

* Install method:

* Clients used: 

* Reverse proxy and version: 

* MySQL/MariaDB or PostgreSQL version: 

* Other relevant details:

**Comment by BlackDex** [comment]
Which version are you running? And which version of your host os are you running?

I suggest to use the Alpine image instead if your host os is not the latest stable version.

**Comment by dani-garcia** [comment]
Sounds like it could be this issue? https://github.com/dani-garcia/vaultwarden/issues/2497

Either using alpine or `--security-opt seccomp=unconfined` solves the problem
